BPOR/BPOR.Registration.Stream.Handler/src/BPOR.Registration.Stream.Handler/Mappers/ParticipantMapper.cs

```diff
@@ -137,18 +149,20 @@ public async Task<Participant> Map(Dictionary<string, AttributeValue> record, Pa
         }
 
         ParticipantAddressMapper.Map(source.Address, destination);
-        var coordinates =
-            await _locationApiClient.GetCoordinatesFromPostcodeAsync(source.Address.Postcode, cancellationToken);
+        // coordinates are not in current stream handler.
+        //var coordinates =
```

Comment:
this is only part of the newer stream handler as it exists in the `rework-rms` branch, from what I can tell this branch has never been deployed ( the stream handler section anyway) , evidenced by the entries in the db, there has been no entires with this location data added to entries since 1st aug this year in prod.
 
I had commented this out as I thought it may be a bit far for this ticket and I didnt want to make too many unrelated changes under this ticket.